packages/edit-site/src/components/global-styles/font-library-modal/utils/preview-styles.js

```diff
@@ -35,9 +35,9 @@ export function formatFontFamily( input ) {
 		.split( ',' )
 		.map( ( font ) => {
 			font = font.trim(); // Remove any leading or trailing white spaces
-			// If the font doesn't have single quotes and contains a space, then add single quotes around it
-			if ( ! font.startsWith( "'" ) && font.indexOf( ' ' ) !== -1 ) {
-				return `'${ font }'`;
+			// If the font doesn't have quotes and contains a space, then wrap in quotes.
+			if ( ! font.startsWith( '"\'' ) && font.indexOf( ' ' ) !== -1 ) {
+				return `"${ font }"`;
 			}
```

Comment:
Check for both `"` and `'` before adding quotes, since either quote type is valid for wrapping font family names.